modules/runner/src/main/java/org/apache/ignite/internal/app/IgnitionImpl.java

```diff
@@ -129,6 +147,24 @@
         return doStart(name, null, workDir);
     }
 
+    /** {@inheritDoc} */
+    @Override public void stop(@NotNull String name) {
```

Comment:
It looks like we need to clearly specify the behavior in the case when a node with the specified name is not found or empty.
Do we need to provide additional clarity for the end-user in this case (return `false` value/ throw a new exception)?

Reply:
Fixed.
```
    /**
     * Stops node with given {@code name}.
     * It's possible to stop both already started node or node that is currently starting.
     * Has no effect if node with specified name doesn't exist.
     *
     * @param name Node name to stop.
     * @throws IllegalArgumentException if null is specified instead of node name.
     */
```